### DISCUSSION TITLE
modernjs官网有白屏报错

### DISCUSSION THREAD

**Discussion by BlInYo** [web-infra-dev/modern.js#5566] [General]
在配置选项下选择左侧的dev开发或者其他的导航会发生以下的错误导致白屏。
Uncaught TypeError: Cannot use 'in' operator to search for 'items' in undefined。

**Comment by BlInYo** [comment]
<img width="1512" alt="Snipaste_2024-03-21_22-51-09" src="https://github.com/web-infra-dev/modern.js/assets/104350815/501e0968-d770-456f-82b3-15db9690788e">

**Reply by 9aoy** [reply]
👍 感谢反馈，目前官网文档已回滚到 2.48.1 版本，修复后我们会重新部署

**Reply by xc2** [reply]
我的 bug 现在还能看到😂

@9aoy rspress 1.15.1 修掉这个问题了

**Reply by 9aoy** [reply]
@BlInYo  官网已更新
